web/modules/custom/portland_migrations/README.md

```diff
@@ -105,6 +105,23 @@ lando terminus remote:drush portlandor.powr-[ID] -- migrate:import category_docu
 lando terminus remote:drush portlandor.powr-[ID] -- migrate:import category_documents_group_content
 ```
 
+### Gotcha
+
+There is an issue when rolling back a migration that depends on another one that interacts with entity revisions. This affects group_content migrations.
+
+For this migration, if the eudaly_news is rolled back and later imported again, the group_content will not be associated with the nodes.
+
+To fix this, the dependent migration eudaly_news_group_content also needs to be rolled back and migrated again. This can be done via the UI or with the following drush commands:
+
+```
+lando drush migrate:rollback eudaly_news_group_content
+lando drush migrate:rollback eudaly_news
+lando drush migrate:import eudaly_news
+lando drush migrate:import eudaly_news_group_content
+```
+
+**Note:** The commands above work for Drush 9. In Drush 8 the command names and aliases are different. Execute `drush list --filter=migrate` to verify the proper commands for your version of Drush.
```

Comment:
Do we need this note? We install Drush 9 via composer on all the environments. Drush 8 should never be needed.